### PR TITLE
E2E: add test assign ip only

### PR DESCRIPTION
**Pull request by liornoy**
As part of an existing test, added a check for the
functionality of assigning ip without announcing.

### CODE REVIEW

**Comment by liornoy** [comment]
The new tests are written under e2etest/bgptests/bgp.go
not sure if it's the right place, as no BGP is involved here.
any suggestions are welcomed.

File: e2etest/bgptests/bgp.go [hunk 1]
```diff
@@ -61,6 +61,63 @@ const (
...
+		defer testservice.Delete(cs, svc)
+
+		for _, i := range svc.Status.LoadBalancer.Ingress {
+			ginkgo.By("validate LoadBalancer IP is in the AddressPool range")
+			ingressIP := e2eservice.GetIngressPoint(&i)
+			err = config.ValidateIPInRange(resources.Pools, ingressIP)
+			framework.ExpectNoError(err)
+		}
+	},
+		table.Entry("IPV4 - ExternalTrafficPolicyCluster", []string{v4PoolAddresses}, testservice.TrafficPolicyCluster),
```
[the remark this answers is not in the document]
Comment (liornoy) [reply e2etest/bgptests/bgp.go:105]: Sounds like a great idea.

[the remark this answers is not in the document]
Comment (liornoy) [reply e2etest/bgptests/bgp.go]: done

**Comment by fedepaol** [comment]
looks good!

**Pull request merged**